components/movementsensor/gpsrtkpmtk/gpsrtkpmtk.go

```diff
@@ -135,7 +133,7 @@ type rtkI2C struct {
 	err          movementsensor.LastError
 	lastposition movementsensor.LastPosition
 
-	nmeamovementsensor gpsnmea.NmeaMovementSensor
+	nmeamovementsensor *rtk.CachedData
```

Comment:
Rename the field to match the new name. I still don't like `CachedData`, and still don't have a better suggestion for what it ought to be. :disappointed: but at least change the field to `cachedData` for now. Same comment in the `rtkSerial` code.